x11rb-protocol/Cargo.toml

```diff
@@ -21,9 +21,17 @@ serde = { version = "1", features = ["derive"], optional = true }
 criterion = "0.5"
 
 [features]
-default = ["std"]
+default = ["extra-traits", "std"]
```

Comment:
Hm.... do we really want this to be enabled by default?

Well... I guess it does not matter for now. :shrug: